sql/values.go

```diff
@@ -37,8 +39,18 @@ func (p *planner) Values(n parser.Values) (planNode, error) {
 		if !ok {
 			return nil, fmt.Errorf("expected a tuple, but found %T", data)
 		}
+		if len(v.rows) == 0 {
+			nCols = len(vals)
+		} else if nCols != len(vals) {
+			return nil, fmt.Errorf("VALUES lists must all be the same length")
+		}
 		v.rows = append(v.rows, vals)
 	}
+
+	v.columns = make([]string, nCols)
+	for i := 0; i < nCols; i++ {
+		v.columns[i] = fmt.Sprintf("column%d", i)
```

Comment:
It gives a `VALUES` statement output column names. I think I accidentally deleted the test case for it. Without column names we pass invalid data back to the client (length of the column names != the number of columns per row).